src/analysis/imports.rs

```diff
@@ -175,7 +175,10 @@ impl Imports {
     ///
     /// Returns `None` if name matches crate name exactly. Otherwise returns
     /// name with crate name prefix stripped or full name if there was no match.
-    fn strip_crate_name<'a>(&self, name: &'a str) -> Option<&'a str> {
+    fn strip_crate_name<'a>(&self, mut name: &'a str) -> Option<&'a str> {
+        if name == "Error" && self.crate_name != "glib" {
+            name = "glib::Error";
```

Comment:
This does not seem correct. Why does it even end up here without the `glib::` prefix?

Reply:
This is actually an excellent question. Since it's part of `manual` imports everytime, it \*should\* not be needed. This fix is therefore incomplete. I need to find what's going on.